### PR TITLE
DF-1545-Events-page-cleanup

### PR DESCRIPTION
**Pull request by sebworks**
Clean up for the events landing and detail page

## Changes

- Update '_event-landing-macros.html' file to clean up line lengths and unnecessary arguments being passed.
- Update 'tags.html" file to simplify logic.
- Update '_event.html' file to remove pad from modification date and add make video link block.
- Update 'index.html' file to add top padding to event list.
- Update 'calendar-icon.less' file to cleanup styles.
- Update 'event.less' file to cleanup styles.
- Update 'modification-date.less' to cleanup styles.

## Review

- @jimmynotjim 
- @anselmbradford

### CODE REVIEW

File: _includes/tags.html [hunk 1]
```diff
@@ -19,19 +19,19 @@
                        using the literal string '/blog/'. Path is used to
                        create the filtered URL: {{ path }}?filter_tags={{ tag }}.
 
-   display_header:     The optional boolean flag to display the tags_header element.
-                       Defaults to True. Adds class 'tags__hide-header' if falsy
-                       value passed (e.g., '', False, None).
+   hide_header:        The optional boolean flag to hide the tags_header element.
+                       Defaults to False. Adds class 'tags__hide-header' if True
+                       value passed (e.g., '', True, None).
```
Comment (anselmbradford) [inline _includes/tags.html]: Is `None` a true value in python?

File: static/css/calendar-icon.less [hunk 2]
```diff
@@ -42,7 +42,7 @@
     background-color: @background-color;
 
     /* Foreground */
-    font-family: 'Avenir Next', Helvetica, Arial;
+    .webfont-regular();
```
Comment (anselmbradford) [inline static/css/calendar-icon.less]: (Referencing) Fixes item in https://github.com/cfpb/cfgov-refresh/issues/296

File: static/css/calendar-icon.less [hunk 3]
```diff
@@ -59,7 +59,7 @@
         text-transform: uppercase;
 
         /* Display & Box Model */
-        padding: 0 0 0.15em;
+        padding-bottom: 0.15em;
```
Comment (anselmbradford) [inline static/css/calendar-icon.less]: (Referencing) Fixes item in https://github.com/cfpb/cfgov-refresh/issues/296

File: static/css/modification-date.less [hunk 4]
```diff
@@ -29,12 +29,16 @@
 */
 
 .modification-date {
+    .h6();
     background: @green-tint;
-    display: inline-block;
+    color: @black;
+    letter-spacing: 0px;
```
Comment (anselmbradford) [inline static/css/modification-date.less]: Can remove units, `letter-spacing: 0`. I believe CSS linters would flag this.

File: _includes/tags.html (hunk 1 above)
Comment (sebworks) [inline _includes/tags.html]: Yep, None is built-in Python constant

File: events/index.html [hunk 5]
```diff
@@ -45,13 +45,14 @@
         <div class="media_body">
             <h1>Upcoming events</h1>
             {# TODO Replace ipsum with latest upcoming event content. #}
-            <p class="h3">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit.<p>
+            <p class="h3 block block__flush-bottom">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit.</p>
```
Comment (jimmynotjim) [inline events/index.html]: Add a semantic line break to avoid horizontal scrolling

File: static/css/modification-date.less [hunk 6]
```diff
@@ -29,12 +29,16 @@
...
+    padding: 1px 10px;
```
Comment (jimmynotjim) [inline static/css/modification-date.less]: Should these be converted to ems with `unit()`?

Comment (sebworks) [inline static/css/modification-date.less]: Yep, will make that change.

File: _includes/tags.html (hunk 1 above)
Comment (anselmbradford) [inline _includes/tags.html]: It doesn't equal True though, right? It's an absence of value I think? So if you pass None as a parameter would it fall back to the default (False)? I'm having trouble running sheer at the moment so can't test.

Comment (sebworks) [inline _includes/tags.html]: Yep, it should default to False if None is passed. Im going to change it to (e.g., '', True, False) which is better example of what is expected.

**Comment by sebworks** [comment]
@anselmbradford, @jimmynotjim changes made based on comments. Please review when you get an opportunity.

**Comment by jimmynotjim** [comment]
:+1:

**Pull request merged**